interactions/api/http/webhook.py

```diff
@@ -119,9 +119,13 @@ async def execute_webhook(
         :return: The message sent, if wait=True, else None.
         """
 
+        dct = {"wait": "true" if wait else "false"}
```

Comment:
Can we please clarify what this is? 😅 I know it's meant to be "dict" but this naming convolutes the concept of this commit.